Marlin/src/MarlinCore.cpp

```diff
@@ -738,6 +738,9 @@ void idle(TERN_(ADVANCED_PAUSE_FEATURE, bool no_stepper_sleep/*=false*/)) {
   // Update the Print Job Timer state
   TERN_(PRINTCOUNTER, print_job_timer.tick());
 
+  // Init buzzer pins
+  TERN_(USE_BEEPER, buzzer.init());
+
```

Comment:
This is in the idle() function, but it should be in setup.

Reply:
Kkkk sorry. It was the hurry to go to bed